spec/responsibility_search_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'spec_helper'
+
+RSpec.describe "Test Search when Responsibility exist" do
+
+	let(:email) {"adneela@hotmail.com"}
+	let(:password) {"adnela"}
+	let(:name) {"Giveee"}
+	let(:ime) {"Adnela Cavcic"}
+	let (:description) {"Descriptiooon"}
+	let(:login) { Login.new(@browser)}
+	let(:responsibility) {Responsibility.new(@browser)}
+
+	context "Login page" do
+	
+        it "Successful login" do
+	       login.login_user(email, password, ime)
+	       @browser.wait_until{@browser.url=="http://ah-test.abhapp.com/profile"}
+		   expect(@browser.text).to include(ime)
+			
+		
+		end
+
+   end
+
+   context "Resonsibilities tab" do
+
+	   it "Switch to Responsibility Page" do
+
+          responsibility.clickon_responsibilities
+          @browser.wait_until{@browser.url == "http://ah-test.abhapp.com/responsibilities"}
+		  expect(@browser.text).to include("Responsibilities")
+	  end
+
+   end
+
+   context "Filter by search" do
+
+   	   it "Searched responsibility stays in list" do
+   		
+   		  responsibility.search(name)
+   		  expect(@browser.text).to include(name)
```

Comment:
You cannot expect to find something that you have not created beforehand. This makes your test depend on data that it does not control. As we already discussed, all data that test uses has to be created in the test before it is checked, and deleted after test finishes.